docs/plugin_development.rst

```diff
@@ -1,34 +1,47 @@
-##############################
-Plugin Development with Spyder
-##############################
+.. _tutorial-main:
 
-This workshop reviews the features and possibilities of the API offered by `Spyder`_ 5—the recently released version of our favorite IDE for scientific Python—for plugin development and to extend its functionality.
+###################################
+Tutorial to develop a Spyder plugin
+###################################
 
-As a practical exercise, we will develop a simple plugin that incorporates a configurable pomodoro timer in the status bar and some toolbar buttons to interact with it.
+This tutorial walks you through creating a Spyder plugin to add a configurable `Pomodoro Technique_` timer to the status bar and toolbar buttons to interact with it.
```

Comment:
```suggestion
This tutorial walks you through creating a Spyder plugin to add a configurable `Pomodoro Technique`_ timer to the status bar and toolbar buttons to interact with it.
```

Fixed syntax error likely introduced in suggestion (despite my double-check before applying)